test/IBCKeeper.t.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.20;
+
+import "forge-std/src/Test.sol";
+import "../src/core/IBCKeeper.sol";
+
+contract DummyHandler {}
+
+contract TestableIBCKeeper is IBCKeeper {
+    constructor(IIBCHandler h) IBCKeeper(h) {}
+
+    function handlerAddr() external view returns (address) {
+        return address(getIBCHandler());
+    }
+}
+
+contract IBCKeeperTest is Test {
+    DummyHandler dummy;
+    TestableIBCKeeper keeper;
+
+    function setUp() public {
+        dummy = new DummyHandler();
+        keeper = new TestableIBCKeeper(IIBCHandler(address(dummy)));
+    }
+
+    function test_GetIBCHandlerReturnsSameAddress() public view {
```

Comment:
There aren’t any strict rules, but the following naming convention might work well.

> function test_add_AddsTwoNumbers() lives within MyContractTest to test the add method.

https://getfoundry.sh/guides/best-practices/writing-tests/#naming-test-files